### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by rmloveland**
Specifically:

+ Use clearer description of the library's purpose (from Prebid.org)
+ Reorder sections based on developer workflow
+ Remove user-facing things that should live on Prebid.org
+ Spellcheck
+ Use (more) consistent names/formatting

### CODE REVIEW

**Comment by rmloveland** [comment]
@mkendall07 here's the PR I promised - might be easier to read as HTML locally (or with `git diff --word-diff` if you wanna get crazy :-) )

File: README.md [hunk 1]
```diff
@@ -3,226 +3,169 @@
...
+
+## Run
+
+To configure Prebid.js to run locally, edit the example file `./integrationExamples/gpt/pbjs_example_gpt.html`:
+
+1. Change `{id}` values appropriately to set up ad units and bidders
+2. Set the path to Prebid.js in your example file as shown below (see `pbs.src`):
+    + For development:
+
+            (function() {
```
Comment (mkendall07) [inline README.md]: looks like we lost our JS formatting here. Can you preface with "```javascript"

File: README.md [hunk 2]
```diff
@@ -3,226 +3,169 @@
...
+
+To view a test coverage report, go to:
+
++ `http://localhost:9999/build/coverage/karma_html/report`
+
+A watch is also in place that will run continuous tests in the terminal as you edit code and tests.
+
+<a name="Contribute"></a>
+
+## Contribute
```
Comment (mkendall07) [inline README.md]: Can you add a section here on unit tests / PRs? Just call out that all PRs require a minimum of 80% unit test code coverage.

**Comment by mkendall07** [comment]
Thanks @rmloveland ! This looks great. Just a few small tweaks I commented on.

**Comment by rmloveland** [comment]
@mkendall07 fixed the JS formatting issue and added a note to the testing section on the >80% testing requirement.  Let me know if you see anything else!

**Comment by mkendall07** [comment]
LGTM. Thanks
